08.ls_object/options_format.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class OptionsFormat
+  def initialize(files)
+    @files = files
+  end
+
+  def list_directory
+    split_list_directory.transpose.each do |split_file|
+      split_file.each { |file| print file.to_s.ljust(30) }
+      print "\n"
+    end
+  end
+
+  private
+
+  COLUMNS_NUMBER = 3
+
+  def create_list_directory
+    column_files = []
+    columns_number = 3
+    @files.each_slice(COLUMNS_NUMBER) do |list|
+      column_files << list
+    end
+
+    column_files.map do |file|
+      file.values_at(0...columns_number)
+    end.flatten
+  end
+
+  def split_list_directory
+    split_files = []
+    create_list_directory.each_slice(create_list_directory.size / COLUMNS_NUMBER) do |list|
+      split_files << list
+    end
+    split_files
```

Comment:
`each_slice` は `Enumerable` を返すので、 `to_a` でもよさそうですね
```suggestion
    create_list_directory.each_slice(create_list_directory.size / COLUMNS_NUMBER).to_a
```
https://docs.ruby-lang.org/ja/latest/method/Enumerable/i/each_slice.html
https://docs.ruby-lang.org/ja/latest/method/Enumerable/i/entries.html

Reply:
参考サイトありがとうございます！ `each`なのでブロックで回すという認識しかなかったです。勉強になりました🙇‍♀️  実装して問題ないことを確認し修正しました！